api/roleTickets/roleTicketsRouter.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router();
+const RoleTickets = require('./roleTicketsModel');
+
+const authRequired = require('../middleware/authRequired');
+const { adminRequired } = require('../middleware/permissionsRequired');
+
+// Responds with all available role ticket requests
+router.get('/', authRequired, adminRequired, (req, res) => {
+  RoleTickets.findAllRoleTickets()
+    .then((roleTickets) => {
+      res.status(200).json(roleTickets);
+    })
+    .catch((err) => {
+      res.status(500).json({ message: err.message });
```

Comment:
I think it would be better to call `next(err)` instead of writing out `res.status(500).json({ message: err.message });`, or maybe change it to just `.catch(next)` -- better to reference the API's main error handler, rather than "hard-coding" a response, directly, to streamline the flow of error messages.

Reply:
I agree with Lance here.